module/zfs/metaslab.c

```diff
@@ -3523,8 +3625,14 @@ metaslab_group_alloc_normal(metaslab_group_t *mg, zio_alloc_list_t *zal,
 			kmem_free(search, sizeof (*search));
 			return (-1ULL);
 		}
-
 		mutex_enter(&msp->ms_lock);
+
+		metaslab_active_mask_verify(msp);
+
+		DTRACE_PROBE3(ms__activation__attempt,
+		    metaslab_t *, msp, uint64_t, activation_weight,
+		    boolean_t, was_active);
```

Comment:
The two new DTRACE_PROBEs are causing the kernel build to fail since the needed tracepoints didn't get added.  We should go ahead and add them (see `include/sys/trace_*.h` for examples).  Alternately, we could comment them out for now since given the work in #8663 there's a good chance we'll want to remove the kernel tracepoint integration entirely quite soon.